DCOS/utils/wsmancmd.py

```diff
@@ -76,7 +77,14 @@ def parse_args():
                 cert_key_pem = arg
             elif opt == "--powershell":
                 is_powershell_cmd = True
-        cmd = args
+            elif opt == "--file":
+                is_powershell_script = True
+        if is_powershell_cmd and not is_powershell_script:
```

Comment:
Let's make a simple check here:

```
if is_script:
    with open(args[0], 'r') as script_file:
        content = script_file.read()
    cmd = content.replace('\n', '; ')
else:
    cmd = args
```

Reply:
Looks like we need another char for one-line commands in `CMD`.

The equivalent of `;` in `CMD` is `&`.